wire-ios-sync-engine/Tests/Source/UserSession/ZMClientRegistrationStatusTests.swift

```diff
@@ -31,4 +31,40 @@ extension ZMClientRegistrationStatusTests {
         // then
         XCTAssertEqual(sut.currentPhase, ZMClientRegistrationPhase.fetchingClients)
     }
+
+    func testThatItNeedsToRegisterMLSClient_WhenNoClientIsAlreadyRegisteredAndAllowed() {
+        // given
+        let selfUser = ZMUser.selfUser(in: syncMOC)
+        selfUser.remoteIdentifier = UUID()
+        DeveloperFlag.enableMLSSupport.enable(true)
+        BackendInfo.apiVersion = .v5
```

Comment:
```suggestion
        DeveloperFlag.storage = .random()!
        DeveloperFlag.enableMLSSupport.enable(true)
        BackendInfo.storage = .random()!
        BackendInfo.apiVersion = .v5
```

Reply:
This avoid these changes form leaking into other tests